lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -32,7 +32,8 @@ def render_to_fragment(self, request, **kwargs):
         Render the program listing fragment.
         """
         user = request.user
-        mobile_only = json.loads(request.GET.get('mobile_only', 'false'))
+        mobile_only = request.GET.get('mobile_only') or 'false'
```

Comment:
Why not doing it `request.GET.get('mobile_only', 'false')`?

Reply:
@ahsan-ul-haq because if someone add `mobile_only` in query param and does not assign the value json.loads() will raise an exception.